packages/collector/test/apps/agentStub.js

```diff
@@ -139,10 +139,14 @@ app.post(
 app.post(
   '/com.instana.plugin.nodejs/traces.:pid',
   checkExistenceOfKnownPid(function handleTraces(req, res) {
+    // eslint-disable-next-line no-console
+    console.log('received spans in agent', req.body);
```

Comment:
~~This will be quite noisy. I'm -1 on adding that permanently, for the whole test suite. Can we remove this before merging?~~ (edit: after reading the whole PR, I guess you are still planning to clean up the console.logs from whole PR anyway)